src/btree/row_srch.c

```diff
@@ -197,6 +200,76 @@ __wt_search_insert(
     cbt->compare = -cmp;
     cbt->ins = (ins != NULL) ? ins : last_ins;
     cbt->ins_head = ins_head;
+
+    /*
+     * FIXME WT-10525 - This is an expensive call so we only want to enabled it behind the
+     * stress_skiplist session flag which will be delivered in WT-10525.
+     * WT_RET(__validate_next_stack(session, cbt->next_stack, srch_key));
```

Comment:
We have a handful of test failures due to `__validate_next_stack`  being an unused function. This'll resolve itself when we add it back in post WT-10525

Reply:
This is indeed fixed now!